src/main/kotlin/app/api/SourcererApi.kt

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app.api
+
+import app.model.Commit
+import app.model.Repo
+import app.model.User
+
+/**
+ * Sourcerer API.
+ */
+object SourcererApi : Api {
+    var api: Api = ServerApi()
```

Comment:
My concern is now we can use SourcererApi directly, e.g. `SourcererApi.postCommits(...)`, if we would use api through variable (getter could be set to variable directly) it would be `SourcererApi.api.postCommits(...)`.

Problems I see:
- It will change existing code where SourcererApi used
- Ugly signature
- Reference of api instance can be saved inside work class (e.g. RepoHasher) and later outside change of api will not affect anything.